### PR TITLE
fix: multiple issues in templates

### PR DESCRIPTION
**Pull request by darahayes**
* Use Mosca in the main showcase template, there is also a showcase template that does have AMQ (Can be manually installed on a project level) https://issues.jboss.org/browse/AEROGEAR-9707
* Use Official Red Hat Node.js Imagestream from the `openshift` namespace - https://issues.jboss.org/browse/AEROGEAR-9693
* Use a HTTPS route that still also allows HTTP traffic - https://issues.jboss.org/browse/AEROGEAR-9708

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,27 +20,25 @@ Prerequisites
...
-* [Data Sync Showcase](https://github.com/aerogear/datasync-deployment/blob/master/openshift/datasync-showcase.yml) - Deploys the server component from the [Ionic Showcase](https://github.com/aerogear/ionic-showcase) application. This includes a postgres storage, a Persistent Volume for file storage, and resources in AMQ Online for subscriptions.
-* [Data Sync Showcase Community](https://github.com/aerogear/datasync-deployment/blob/master/openshift/datasync-showcase-community.yml) - Deploys the server component from the [Ionic Showcase](https://github.com/aerogear/ionic-showcase) application , but replaces Red Hat AMQ Online with [Mosquitto](https://mosquitto.org/) as the MQTT broker.
+* [Data Sync Showcase](https://github.com/aerogear/datasync-deployment/blob/master/openshift/datasync-showcase.yml) - Deploys the server component from the [Ionic Showcase](https://github.com/aerogear/ionic-showcase) application. This includes a postgres storage, a Persistent Volume for file storage, and a Mosca MQTT Broker for a quick and simple getting started experience.
```
Comment (wtrocki) [inline README.md]:
```suggestion
* [Data Sync Showcase](https://github.com/aerogear/datasync-deployment/blob/master/openshift/datasync-showcase.yml) - Deploys the server component from the [Ionic Showcase](https://github.com/aerogear/ionic-showcase) application. This includes a postgres storage, a Persistent Volume for file storage, and a Mosquitto MQTT Broker for a quick and simple getting started experience.
```

**Review by wtrocki — APPROVED** [review]

File: openshift/datasync-showcase.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ labels:
 metadata:
   name: datasync-showcase-server
   annotations:
-    openshift.io/display-name: Data Sync Showcase 
+    openshift.io/display-name: Data Sync Showcase (test)
```
Comment (wtrocki) [inline openshift/datasync-showcase.yml]:
```suggestion
    openshift.io/display-name: Data Sync Showcase
```

**Comment by darahayes** [comment]
@wtrocki you can review those changes here

showcase no amq - https://master.mdsdemo-710a.openshiftworkshop.com/console/project/dhayes-showcase-new/overview

showcase with amq - https://master.mdsdemo-710a.openshiftworkshop.com/console/project/dhayes-showcase-new-amq/overview

**Comment by wtrocki** [comment]
Done

**Pull request merged**